code/game/objects/objs.dm

```diff
@@ -196,6 +197,27 @@
 	..()
 	if(unique_rename)
 		to_chat(user, "<span class='notice'>Use a pen on it to rename it or change its description.</span>")
+	if(unique_reskin && !current_skin)
+		to_chat(user, "<span class='notice'>Alt-click it to reskin it.</span>")
+
+/obj/AltClick(mob/user)
+	. = ..()
+	if(unique_reskin && !current_skin && in_range(user,src))
+		if(user.incapacitated())
+			to_chat(user, "<span class='warning'>You can't do that right now!</span>")
+			return
+		reskin_obj(user)
+
+/obj/proc/reskin_obj(mob/M)
+	if(!unique_reskin || !islist(unique_reskin))

```

Comment:
actually, he needs to check that it is a list, because LAZYLEN(<string>) returns a non-zero